Dockerfile

```diff
@@ -50,10 +50,12 @@ COPY . .
 RUN git submodule update --init --recursive
 
 FROM conda as build
+ARG CMAKE_VARS
 WORKDIR /opt/pytorch
 COPY --from=conda /opt/conda /opt/conda
 COPY --from=submodule-update /opt/pytorch /opt/pytorch
 RUN --mount=type=cache,target=/opt/ccache \
+    export eval ${CMAKE_VARS} && \
```

Comment:
It seems that `eval` with empty `${CMAKE_VARS}` works ok... You lucky bastard! ;)